draft-ietf-mops-streaming-opcons.md

```diff
@@ -739,6 +739,8 @@ This document introduces no new security issues.
 
 # Acknowledgments
 
-Thanks to Mark Nottingham, Glenn Deen, Dave Oran, Aaron Falk, Kyle Rose, Leslie Daigle, Lucas Pardue, Matt Stock, Alexandre Gouaillard, and Mike English for their very helpful reviews and comments.
+Thanks to Alexandre Gouaillard, Aaron Falk, Dave Oran, Glenn Deen, Kyle Rose, Leslie Daigle, Lucas Pardue, Mark Nottingham, Matt Stock, Mike English, and Roni Even for very helpful suggestions, reviews and comments.
```

Comment:
Now that I'm reading it, I think we got some good comments from Will Law too, he should probably be in here.

Reply:
Added now!